MapboxAndroidDemo/src/main/res/values/strings.xml

```diff
@@ -81,6 +81,7 @@
     <string name="activity_location_tracking_title">Track user location</string>
     <string name="activity_location_animated_icon_title">Animate the user icon</string>
     <string name="activity_location_customize_user_title">Customize user location icon</string>
+    <string name="activity_location_turf_inside_title">Use Java Turf</string>
```

Comment:
Can we name the title something along the lines of `turf inside`. Java Turf is too generic for this example